packages/webcomponents/src/components/business-details/generic-info-details/generic-info-details.tsx

```diff
@@ -17,34 +17,47 @@ export class GenericInfoDetails {
         <DetailSection sectionTitle="Business Generic Info Details">
           <div class="row gy-3">
             <div class="col-12 col-md-6">
-              <DetailItem
-                title="Legal Name"
-                value={this.business?.legal_name}
-              />
-              <DetailItem
-                title="Doing Business As (DBA)"
-                value={this.business?.doing_business_as}
-              />
-              <DetailItem
-                title="Business Type"
-                value={snakeCaseToHumanReadable(this.business?.business_type)}
-              />
-              <DetailItem
-                title="Business Structure"
-                value={snakeCaseToHumanReadable(
-                  this.business?.business_structure,
-                )}
-              />
-              <DetailItem title="Industry" value={this.business?.industry} />
+              {this.business?.legal_name && (
```

Comment:
I'm not sure if these conditionals are needed here for the `DetailItem` components - inside `DetailItem` contains a ternary that renders the component OR null based on the presence of `title` and `value`

```
export const DetailItem = ({
  title,
  value,
}: {
  title: string;
  value: string;
}) =>
  title && value ? (
    <div class="d-table-row gap-2">
      <span part="detail-section-item-title" class="fw-bold d-table-cell px-2">
        {title}
      </span>
      <span
        part="detail-section-item-data"
        class="flex-1 d-table-cell px-2 text-wrap"
        innerHTML={value}
      ></span>
    </div>
  ) : null;
```

So maybe we can get away with doing this without these checks? What do you think?